client/lib/polyfills.js

```diff
@@ -88,9 +88,41 @@ function fillPolyfill () {
   }
 }
 
+function findIndexPolyfill () {
+  if (!Array.prototype.findIndex) {
+    Object.defineProperty(Array.prototype, 'findIndex', {
+      value: function(predicate) {
+        'use strict';
+        if (this == null) {
+          throw new TypeError('Array.prototype.findIndex called on null or undefined');
+        }
+        if (typeof predicate !== 'function') {
+          throw new TypeError('predicate must be a function');
+        }
+        var list = Object(this);
+        var length = list.length >>> 0;
```

Comment:
I don't _understand_ why the result of calling the `.length` method would _not_ be a Number, but I'm not _offended_ by this syntax.